scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -598,6 +598,12 @@ static bool match_one(struct layer_match *match, struct task_struct *p, const ch
 		memcpy(comm, p->comm, MAX_COMM);
 		return match_prefix(match->comm_prefix, comm, MAX_COMM);
 	}
+	case MATCH_PNAME_PREFIX: {
```

Comment:
Would `PCOMM` be a name which is more consistent with `COMM`?

Reply:
Agreed, done